mitiq/shadows/rotation_gates.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
+
+import cirq
+import numpy as np
+
+
+# generate N random Pauli strings for given number of qubits
+def generate_random_pauli_strings(
+    num_qubits: int, num_strings: int
+) -> List[str]:
+    # return "XXXYZYZX" where len string == num_qubits
```

Comment:
Please add a docstring using the Google-style format used also in other functions (such as `get_rotated_circuits`).